src/gConsent/manage/approveAccessRequest.ts

```diff
@@ -108,7 +116,9 @@ async function addVcMatcher(
         );
       }
       // eslint-disable-next-line camelcase
-      const updatedResource = acp_ess_2.setVcAccess(resourceInfo, accessMode);
+      const updatedResource = acp_ess_2.setVcAccess(resourceInfo, accessMode, {
+        inherit: options?.inherit ?? true,
```

Comment:
We're defaulting this to true now?

Reply:
Indeed: the Access Grants are defaulting to recursive, so the ACR structure should do the same otherwise recursive Access Grants aren't applicable.

Reply:
Is that how product is rolling this out? Shouldn't they default to non-recursive in this major release (as was previously the case), and then on the next semver major we make them recursive by default?

Reply:
Everyone we talked to tripped over the current behavior, expecting granting access to a container to also apply on contained resources. So technically yes, but here it's more aligning with expectations.